code/game/objects/items/reagent_containers/food/snacks.dm

```diff
@@ -1483,7 +1483,22 @@
 	icon_state = "barcardine"
 	wrapper = /obj/item/trash/barcardine
 	list_reagents = list(/datum/reagent/consumable/nutriment = 3, /datum/reagent/consumable/coco = 2, /datum/reagent/medicine/tramadol = 1, /datum/reagent/medicine/tramadol = 1)
-	tastes = list("cough syrup" = 1)
+	tastes = list ("cough syrup" = 1)
+
+/obj/item/reagent_containers/food/snacks/wrapped/berrybar
+	name = "Berry Bars"
```

Comment:
```suggestion
	name = "Health Bars"
```
do it coward

Reply:
![image](https://github.com/tgstation/TerraGov-Marine-Corps/assets/131525549/373b5739-561a-4fbe-b14b-c11599b46cea)

Reply:
Dont do it toki stay strong 🙏